recipe/meta.yaml

```
@@ -1,30 +1,25 @@
 {% set name = "psutil" %}
-{% set version = "5.9.0" %}
-{% set sha256 = "869842dbd66bb80c3217158e629d6fceaecc3a3166d3d1faee515b05dd26ca25" %}
+{% set version = "7.0.0" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/p/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7be9c3eba38beccb6495ea33afd982a44074b78f28c434a1f51cc07fd315c456
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install --ignore-installed --no-deps .
-  # we currently do not support python versions earlier than 3.7
-  skip: true  # [py<37]
+  number: 0
+  skip: true  # [py<36]
```

```suggestion
build:
  number: 0
  script:
    # macOS renamed this variable, but it is `NULL` in either case.
    # Workaround the missing definition by setting it to `0`.
    #
    # xref: https://github.com/giampaolo/psutil/issues/2354
    - export CFLAGS="${CFLAGS} -DkIOMainPortDefault=0"  # [osx]
    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
```
The feedstock can be simpler if you just do the export in the recipe instead of making build scripts for each platform.